src/main/java/entities/CollaborativeTask.java

```diff
@@ -0,0 +1,108 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class CollaborativeTask extends Task implements Timeblockable{
+    private boolean recurring;
+    private String frequency;
+    private ArrayList<StudentUser> teammates;
+    private ArrayList<StudentUser> pendingTeammates;
+    private ArrayList<StudentUser> declinedTeammates;
+    private StudentUser leader;
+
+    /**
+     * Create a CollaborativeTask with a title
+     * If the collaborative task is recurring, indicate the frequency (eg "daily", "weekly", "monthly)
+     * Otherwise, the frequency is blank
+     * @param title - the title of the Collaborative Task
+     * @param recurring - whether the Collaborative Task is recurring
+     * @param frequency - the frequency at which the Collaborative Task occurs (if recurring)
+     */
+    public CollaborativeTask(String title, boolean recurring, String frequency, StudentUser creator) {
```

Comment:
So sorry but since I changed Task slightly the constructor needs a String id parameter after title for both constructors, and change super(title) to super(title, id)